src/sparseml/pytorch/utils/model.py

```diff
@@ -241,7 +242,7 @@ def save_model(
     if arch_key:
         save_dict["arch_key"] = arch_key
 
-    if torch.__version__ < "1.6":
+    if version.parse(torch.__version__) < version.parse("1.6"):
```

Comment:
why are we doing it the more complicated way?

Reply:
Should never just compare version directly because it's doing string comparison. e.g. `"1.10.0" < "1.9.0"` is true